ui/packages/consul-ui/app/locations/fsm-with-optional.js

```diff
@@ -254,6 +254,10 @@ export default class FSMWithOptionalLocation {
    * performs an ember transition/refresh and browser location update using that
    */
   transitionTo(url) {
+    if (typeof this.router === 'undefined') {
+      this.router = this.container.lookup('router:main');
+    }
```

Comment:
Mostly curious was this due to the change you made?
Or something that you just noticed along the way?

Reply:
I just noticed that there was issues when trying to navigate to Docfy routes (eng docs). Sometimes the router would be null. Before we used Docfy generated link components but I wanted to just use the side nav stuff.